qiita_ware/ebi.py

```diff
@@ -161,7 +161,7 @@ def __init__(self, artifact_id, action):
         self.experiment_xml_fp = None
         self.run_xml_fp = None
         self.submission_xml_fp = None
-        self.pmids = self.study.pmids
+        self.publications = self.study.publications
```

Comment:
This is a good point. It's not really redundant as self.publications will be a list of the publications and self.study.publications is a query to the DB to retrieve the publications. I could remove but I like the idea that in the init we make sure that everything that is possible looks sane.